frontend/src/js/controllers/authCtrl.js

```diff
@@ -117,8 +133,15 @@
                     }
                 }
             };
+            var passwordMatch = vm.passwordCheck();
```

Comment:
call this something like 
i.e var passwordMatch = vm.passwordCheck(password1, password2)